### DISCUSSION TITLE
Style CI CLI Tool Task

### DISCUSSION THREAD

**Discussion by babul** [phpro/grumphp#899] [Ideas]
| Q               | A
| --------------- | ---
| Version         | GrumPHP 0.20.0
| Bug?            | no
| New feature?    | yes
| Question?       | yes
| Documentation?  | no
| Related tickets | 

Before we start to make a custom task, has anyone built a custom task for the StyleCI CLI Tool (https://docs.styleci.io/cli-tool)?

Thanks!

**Comment by veewee** [comment]
Hello @babul ,

We currently don't have that task built in. 
Feel free to provide a PR.
